test/scpi.py

```diff
@@ -93,14 +94,89 @@ def get_clk_rate(classname, iio_uri):
     return float(clk_rate)
 
 
+def save_to_eeprom(iio_uri, coarse, fine):
+    full_uri = iio_uri.split(":", 2)
+    if full_uri[0] != "ip":
+        print("Tuning currently supported only for ip URIs")
```

Comment:
Should this generate an exception?

Reply:
I don't know if it should generate an exception necessarily, maybe a pytest.skip is more appropriate?

Reply:
sure that works too